src/asm/lexer.c

```diff
@@ -1934,37 +2073,60 @@ static int yylex_RAW(void)
 			return T_STRING;
 
 		case '\\': /* Character escape */
-			c = peek(1);
+			shiftChars(1); /* Shift the backslash */
+			c = peek(0);
+
 			switch (c) {
+			case ';': /* Macro-only character escapes */
```

Comment:
See discussion about whether to allow `\;`.

Reply:
I'm in favor of allowing `\;` for completeness: for instance, if someone wants to write a macro that only does `"\1"`, never bare `\1`. Also, the same sort of issue exists for block comments: you can do some tricky and weird things by putting `/*` and `*/` within `if/else` bodies, or as macro args.

Reply:
Removed support for `\;`.